templates/page.html

```diff
@@ -176,20 +172,19 @@ <h3>Run your analysis</h3>
         <figure class="figure">
           <div class="command-line">
             <div class="line comment"># install reana-client</div>
-            <div class="line"><span class="prompt">$</span>mkvirtualenv reana-client -p /usr/bin/python2.7</div>
+            <div class="line"><span class="prompt">$</span>mkvirtualenv reana-client</div>
             <div class="line"><span class="prompt">$</span>pip install reana-client</div>
             <div class="line"><span class="prompt">$</span>reana-client ping</div>
             <div class="line comment"># create new workflow</div>
-            <div class="line"><span class="prompt">$</span>export REANA_WORKON=$(reana-client workflow create)</div>
-            <div class="line comment"># upload runtime code and inputs</div>
-            <div class="line"><span class="prompt">$</span>reana-client code upload ./code/*</div>
-            <div class="line"><span class="prompt">$</span>reana-client inputs upload ./inputs/*</div>
+            <div class="line"><span class="prompt">$</span>export REANA_WORKON=$(reana-client create)</div>
```

Comment:
For the workflow creation, what about using:

```console
$ reana-client create -n my-analysis
$ export REANA_WORKON=my-analysis
```

May be more readable, see https://github.com/reanahub/reana/pull/87

Let's harmonise amongst the two PRs...

Reply:
I am on it!